### ISSUE TITLE
On card click is not triggered any more

### ISSUE THREAD

**Issue by julienpillias** [rcdexta/react-trello#56]
Hi, 

With last update (1.26.1), onCarcClick i snot triggered any more : 

```js
<Board
              draggable={true}
              handleDragEnd={(cardId, sourceLaneId, targetLaneId) => this.dragCardToCol(cardId, targetLaneId)}
              tagStyle={{fontSize: '70%'}}
              onCardClick={(cardId, metadata, laneId) => alert(cardId)}
              onCardAdd={(card, laneId) => this.addCard(card, laneId)}
              onCardDelete={(cardId, laneId) => this.askDeleteCard(cardId, laneId)}
              newCardTemplate={<NewCard />}
              customCardLayout
              data={data}
              addCardLink={
                <a href="#">
                  <i className="fa fa-plus" aria-hidden="true"></i>
                  &nbsp;Ajouter une tâche
                </a>
              }
              editable
            >
              <CustomCard/>
``` 

When I click on the card, the alert is never displayed (same with a custom function this.showDetails(cardId) )

**Comment by julienpillias** [comment]
more information : 
Tried with version 1.26.0 => all fine
version 1.26.1 => can't click

**Comment by rcdexta** [comment]
Fixed in 1.26.2. Thanks and sorry for the inconvenience!
